kaldb/src/main/java/com/slack/kaldb/config/KaldbConfig.java

```diff
@@ -25,6 +26,14 @@
   public static Duration DEFAULT_START_STOP_DURATION = Duration.ofSeconds(15);
   public static final int DEFAULT_ZK_TIMEOUT_SECS = 15;
 
+  // Timeouts are structured such that we always attempt to return a successful response, as we
+  // include metadata that should always be present.
+  public static final Duration ARMERIA_TIMEOUT = Duration.of(15, ChronoUnit.SECONDS);
```

Comment:
Renaming `ARMERIA_TIMEOUT` to `ARMERIA_QUERY_TIMEOUT_SECS` is more consistent with other fields. Also, please add time units to other timeout values.

Please expand comment explaining where various timeouts are used and why different timeouts are smaller than others.

Reply:
Done. Per discussion used `_DURATION` for all these fields and added the additional documentation.